### ISSUE TITLE
We need the ability to copy text (with a checkbox in the widget settings that enables/disables this feature)

### ISSUE THREAD

**Issue by DAVIDhaker** [Zren/plasma-applet-commandoutput#46]

**Comment by davidosomething** [comment]
fwiw, you can pipe command output in `click`, e.g.

for my custom tailscale widget, i have the click action set to
```
ip=$(tailscale ip --4 2>/dev/null) && echo "$ip" | xsel --clipboard && notify-send "Copied ${ip}"
```

which copy my tailscale ip to the clipboard using xsel, and send a desktop notification using notify-send

**Comment by DAVIDhaker** [comment]
@davidosomething, yes. Good idea, but how to be, when i need to copy random parts/string from text?

**Comment by davidosomething** [comment]
grep to match the pattern and pipe the grep results, you can find grep commands for matching urls on stackoverflow for instance
